cmake/KernelFunctions.cmake

```diff
@@ -3,19 +3,28 @@ function(add_kernel name)
 endfunction()
 
 set(common_dpcpp_libs common dpcpp_common sycl oclhelpers::oclhelpers)
+set(common_tbb_libs common tbb)
```

Comment:
This didn't work out for me with a clean environment. OneTBB examples show that proper linking should be done as TBB::tbb. See https://github.com/oneapi-src/oneTBB/blob/master/examples/parallel_for/game_of_life/CMakeLists.txt#L26 for example.
With the fix, works fine.